website/docs/r/s3_bucket_replication_configuration.html.markdown

```diff
@@ -201,9 +200,9 @@ aws_s3_bucket_replication_configuration "west_to_east" {
 
 ~> **NOTE:** To avoid conflicts always add the following lifecycle object to the `aws_s3_bucket` resource of the source bucket.
 
-This resource implements the same features that are provided by the `replication_configuration` object of the `aws_s3_bucket` resource.  To avoid conflicts or unexpected apply results a lifecycle configuration is needed on the `aws_s3_bucket` to ignore changes to the internal `replication_configuration` object.  Faliure to add the `lifecycle` configuation to the `aws_s3_bucket` will result in conflicting state results.
+This resource implements the same features that are provided by the `replication_configuration` object of the `aws_s3_bucket` resource.  To avoid conflicts or unexpected apply results a lifecycle configuration is needed on the `aws_s3_bucket` to ignore changes to the internal `replication_configuration` object.  Failure to add the `lifecycle` configuration to the `aws_s3_bucket` will result in conflicting state results.
 
-```terraform
+```hcl
```

Comment:
giving this type a try since `terraform` type expects valid configurations

Reply:
@dkujawski best bet for now might be to leave these as untyped code blocks since `tflint` will still try to validate the `hcl`